### PR TITLE
New DigitalOcean Spaces Profiles

### PR DESCRIPTION
**Pull request by jbraunstein-do**

### CODE REVIEW

File: DigitalOcean Spaces (blr1).cyberduckprofile [hunk 1]
```diff
@@ -0,0 +1,558 @@
...
+        <key>Regions</key>
+        <array>
+            <string>blr1</string>
+        </array>
+        <key>Properties</key>
+        <array>
+            <string>s3.storage.class.options=STANDARD</string>
+        </array>
+        <key>Help</key>
+        <string>https://trac.cyberduck.io/wiki/help/en/howto/digitalocean</string>
```
Comment (dkocher) [inline DigitalOcean Spaces (blr1).cyberduckprofile:51]: Please change to the redirect target `https://docs.cyberduck.io/protocols/s3/digitalocean/`

```suggestion
        <string>https://docs.cyberduck.io/protocols/s3/digitalocean/</string>
```

File: DigitalOcean Spaces (syd1).cyberduckprofile [hunk 2]
```diff
@@ -0,0 +1,558 @@
...
+        <key>Regions</key>
+        <array>
+            <string>syd1</string>
+        </array>
+        <key>Properties</key>
+        <array>
+            <string>s3.storage.class.options=STANDARD</string>
+        </array>
+        <key>Help</key>
+        <string>https://trac.cyberduck.io/wiki/help/en/howto/digitalocean</string>
```
Comment (dkocher) [inline DigitalOcean Spaces (syd1).cyberduckprofile:51]: Please change to the redirect target `https://docs.cyberduck.io/protocols/s3/digitalocean/`

```suggestion
        <string>https://docs.cyberduck.io/protocols/s3/digitalocean/</string>
```

File: DigitalOcean Spaces (sfo3).cyberduckprofile [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
```
Comment (dkocher) [inline DigitalOcean Spaces (sfo3).cyberduckprofile:1]: Can be omitted in this changeset.

**Review by dkocher — CHANGES_REQUESTED** [review]

**Review by dkocher — APPROVED** [review]

**Comment by jbraunstein-do** [comment]
Done, thank you. https://github.com/iterate-ch/docs/pull/404

**Pull request merged**